setup.cfg

```
@@ -25,6 +25,7 @@ packages=find:
 include_package_data = true
 zip_safe = false
 install_requires =
+    backports.os == 0.1.1
```

IMHO this should be only for Python 2 as in https://github.com/nexB/scancode-toolkit/blob/f604cf45270de9afff9fcfacf97f2124b9ca9fcd/setup.py#L154

```suggestion
    backports.os == 0.1.1; python_version < "3"
```

---

Good point I was getting syntax errors, so I removed it. But it works now with the above changes, so will commit those.